### PR TITLE
add in game force display dialog

### PR DESCRIPTION
**Pull request by kuronekochomusuke**
- add in game force display dialog, similar to the lobby force display 
- double clicking a unit views it in unit display and centers unit on game board
- popup menu for view readout of unit
- add an indicator if a unit is done or not
- show when a unit is destroyed

image
image

image

### CODE REVIEW

**Comment by HammerGS** [comment]
Been playing around with this, very cool feature and have found no issues with it. But testing has been limited a smaller scale.

**Comment by HammerGS** [comment]
Wanted to confirm if this is still in development or ready to go?

**Comment by kuronekochomusuke** [comment]
this is ready to review

File: megamek/src/megamek/client/ui/swing/GUIPreferences.java [hunk 1]
```diff
@@ -129,6 +129,14 @@ public class GUIPreferences extends PreferenceStoreProxy {
     public static final String CUSTOM_UNIT_HEIGHT = "CustomUnitDialogSizeHeight";
     public static final String CUSTOM_UNIT_WIDTH = "CustomUnitDialogSizeWidth";
 
+    public static final String FORCE_DISPLAY_POS_X = "ForceDisplayPosX";
+    public static final String FORCE_DISPLAY_POS_Y = "ForcetDisplayPosY";
+    public static final String FORCE_DISPLAY_SIZE_HEIGHT = "ForceDisplaySizeHeight";
+    public static final String FORCE_DISPLAY_SIZE_WIDTH = "ForceDisplaySizeWidth";
+    public static final String FORCE_DISPLAY_AUTO_DISPLAY_REPORT_PHASE = "ForceDisplayAutoDiplayReportPhase";
+    public static final String FORCE_DISPLAY_AUTO_DISPLAY_NONREPORT_PHASE = "ForceDisplayAutoDiplayNonReportPhase";
```
Comment (Sleet01) [inline megamek/src/megamek/client/ui/swing/GUIPreferences.java:137]: Typo: `ForceDisplayAutoDiplay` -> `ForceDisplayAutoDisplay`

**Review by Sleet01 — CHANGES_REQUESTED** [review]
Just a couple of minor typos, otherwise looks good.

**Pull request merged**